helper/ThemeFunctions.php

```
@@ -0,0 +1,1342 @@
+
+        // Renderable.
+        // To manage: image, audio, video, html, pdf, office, model, iiif, score…
+        return $viewing;
+    }
+
+    /**
+     * Get the medias of an item according to media type, renderer and viewer.
+     *
+     * @deprecated Use module BlockPlus.
+     */
+    public function listMediasByViewer(ItemRepresentation $item, ?string $viewer = null): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$viewer'. (Unused Code Rules)

(UnusedFormalParameter)